appdev/guides/pushnotifications.rst

```diff
@@ -0,0 +1,235 @@
+Push notifications
+==================
+
+Let's assume that you have an app created with Clickable and published on the OpenStore. But now you want to be able to send Push Notifications to your users. First of all, you need to understand how this is working:
+
+.. image:: ../../_static/images/appdev/guides/pushnotifications/Diagram.png
+
+1. Every app which uses push notifications has got a unique token. This token identifies the user, the device and the app itself. The token is generated by the UBports Push Service.
+2. You will need the token to send a push notification so the app needs to send this token to the entity, which will send the notification.
+3. With the token you can send a simple HTTP request to the UBports Push Server which will forward the notification the user's device.
```

Comment:
As a style change, I prefer to not use "just," "simple," "easy," or "clear" when referring to actions taken from our documentation. They might not be easy or clear for some people.

* https://dev.to/marcuscreo/the-4-letter-word-word-that-makes-my-blood-boil
* https://www.glamour.com/story/why-i-quit-using-the-word-just-in-my-emails

That said, I need to put in a PR to remove it from other places.

Reply:
Okay, I have removed these words.